remoteworkitem/remoteworkitem.go

```diff
@@ -22,6 +22,12 @@ var WorkItemKeyMaps = map[string]WorkItemMap{
 		AttributeExpression("state"): SystemStatus,
 		AttributeExpression("id"):    SystemRemoteItemId,
 	},
+	ProviderJira: WorkItemMap{
+		AttributeExpression("fields.summary"):     SystemTitle,
+		AttributeExpression("fields.description"): SystemDescription,
+		AttributeExpression("fields.status.name"): SystemStatus,
```

Comment:
Thanks for pointed out, updated them in the latest commit.
Note: Assignee is an optional key in both Jira and GH.